src/App.jsx

```diff
@@ -3,6 +3,7 @@ import './App.css';
 import logo from './logo.svg';
 
 function App() {
+  document.title = 'Spotties';
```

Comment:
all lowercase, please 😘

Reply:
Also, do you think that it would be helpful to have a `setPageTitle(newTitle)` function that can be used from any component, or is it overengineering?